lib/en/managing-markers.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: base
+category: 17_Managing Your Applications
+breadcrumb: Managing Your Applications
+parent_url: managing-overview.html
+nav_title: Markers
+nav_priority: 82
+meta_desc: Control your app using markers.
+---
+= Markers
+
+[[top]]
+[float]
+= Markers
+[.lead]
+Marker files are used to set conditions for various stages of a cartridge's life cycle, enabling you to control certain aspects of the cartridge.
+
+
+[[using-markers]]
+== Using markers
+Enabling a marker is as simple as creating an empty file of a given name in the `.openshift/markers/` directory and link:managing-modifying-applications.html[pushing] it to your repository, in order to trigger the required functionality.
+
+TIP: For example, creating the `.openshift/markers/hot_deploy` in an PHP app will prevent the Apache process from being restarted during the link:managing-deployments.html[deployment].
+
+Creating an empty file in the current directory is typically done using `touch marker_file` on Linux and Mac OS systems or by `copy NUL > marker_file` when on Windows.
+
+[[marker-files-overview]]
+== Marker files overview
+
```

Comment:
Some markers are general, such has `hot_deploy` and `force_clean_build`. How about adding a table here with the generic markers?